salsa_dancing_molecules/calculator/capacity_NVE.py

```diff
@@ -14,9 +14,12 @@
 
 def get_NVE_heat_capacity(a):
     """Calculat Heat Capacity for a NVE ensamble."""
+    u_to_kg = sum(a.get_masses())*1.6605*10**(-27)
```

Comment:
Can we find these constants in the ASE units module?

This goes for the other file as well.